### ISSUE TITLE
.NET 7 gitignore template miss .idea (JetBrains Rider) exclusion

### ISSUE THREAD

**Issue by elendil-software** [dotnet/sdk#29224]
### Describe the bug

In .NET 6, the gitignore template was containing the following JetBrains Rider exclusions 

```
# JetBrains Rider
.idea/
*.sln.iml
```

the .NET 7 template only contains 

```
# JetBrains Rider
*.sln.iml
```

the .idea/ exclusion is missing.

### To Reproduce

call `dotnet new gitignore` using .NET 7 SDK, the generated file miss the .idea/ exclusion

**Comment by elendil-software** [comment]
Rethinking about that, it make sense to have removed .idea from the VisualStudio exclusions. But shouldn't *.sln.iml have been removed too ?

I looked a bit more into the .idea directory structure and I found that having some files in a VCS can be something that is wanted. Therefore, the .idea folder should not be excluded by default by a generated .gitignore.

https://rider-support.jetbrains.com/hc/en-us/articles/207097529-What-is-the-idea-folder-

I also found out that JetBrains IDEs create their own .gitignore files inside .idea directory structure. So by default there probably nothing to do.
